javascript/javascript1/week4/palindromic_string/pstring.js

```diff
@@ -0,0 +1,32 @@
+function longestPalindrome(str) {
+  const string = str.toLowerCase();
```

Comment:
ouch, don't name anything `string`, that is a special keyword that signifies the `string` type.

Reply:
In this case str=str.toLowerCase() would work just fine.